interview-questions.md

```diff
@@ -8,36 +8,66 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. As a developer, I am creating a Rails application with a model called Cohort and a model called Student. The cohort model has_many students, but OOPS! I forgot to add the foreign key. How can I fix this mistake? What is the name of the foreign key? Would the foreign key be on the Cohort model or the Student model?
 
-Your answer:
+Your answer: The name of the foreign key is cohort_id followed by a integer. You can fix the mistake by adding it to the migration file. The key would be in the cohort model. 
 
-Researched answer:
+Researched answer: You can run the command rake db:rollback, then go to migration and edit foreign key.
```

Comment:
db:rollback will reverse the most recent database migration. That being said, if the creation of the Student model is not the most recent migration, you will need to find a different method.